container/k8s/base_deploy.py

```diff
@@ -348,24 +368,31 @@ def _service_to_container(name, service):
                     container['workingDir'] = value
                 else:
                     container[key] = value
+            return container, volumes
 
-                print(container)
+        templates = CommentedSeq()
+        for name, service_config in iteritems(self._services):
+            containers = []
+            volumes = []
+            pod = {}
+            if service_config.get('containers'):
+                for c in service_config['containers']:
+                    cname = "{}-{}".format(name, c['container_name'])
+                    k8s_container, k8s_volumes, = _service_to_k8s_container(name, c, container_name=cname)
+                    containers.append(k8s_container)
+                    volumes += k8s_volumes
```

Comment:
This can lead to duplicate volumes if two containers are mounting the same volume
```
spec.template.spec.volumes[1].name: Duplicate value: \"static-content\""
```